tracker_test.go

```diff
@@ -1,63 +1,29 @@
-// +build integration
-
 package main
 
 import (
 	"testing"
 
-	"github.com/almighty/almighty-core/app"
+	"golang.org/x/net/context"
+
```

Comment:
@baijum I think, this is what `go fmt` likes to do: Separate standard from 3rd party libs via an external line. I noticed it when I formatted my code as well.